web/massastation/src/pages/Store/Sidepanel.tsx

```diff
@@ -0,0 +1,85 @@
+import { Button, Input, SidePanel } from '@massalabs/react-ui-kit';
+import Intl from '../../i18n/i18n';
+import { useEffect, useState } from 'react';
+import { parseForm } from '../../utils/ParseForm';
+import { regexZipCheck } from '../../utils/massaConstants';
+
+// no camelcase SidePanel here because it will create conflicts
+function Sidepanel() {
+  const [url, setUrl] = useState<string>('');
```

Comment:
do we need this ? can't we just get the value from the form ?

Reply:
we also need to be able to re-inject the value from the form as the value of the input because e.target.value doesn't work